test/stresstest/server.py

```diff
@@ -0,0 +1,54 @@
+########################################################################
+# Copyright (c) 2022 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+########################################################################
+
+import subprocess
+import os.path
+import config
+import signal
+import argparse
+
+class Server:
+    def build(self):
+        isdir = os.path.isdir(self.build_path)
+        if not isdir:
+            subprocess.check_call(f"mkdir {self.build_path}")
```

Comment:
```suggestion
       os.makedirs(self.build_path,  exist_ok=True)
```
Does the same more or less without requiring a subprocess.